rollup.config.ts

```diff
@@ -2,6 +2,7 @@ import commonjs from '@rollup/plugin-commonjs';
 import resolve from '@rollup/plugin-node-resolve';
 import { terser } from 'rollup-plugin-terser';
 import typescript from '@rollup/plugin-typescript';
+
```

Comment:
delete this line

Reply:
It's intentional. I added this line to separate the two types of import groups.